docs/embedding.md

```diff
@@ -289,15 +294,15 @@ def embed_dataset():
   sample_size = int(ttl_size * 0.01) # We work with 1% of the dataset 
   subset = dataset["train"].select(range(sample_size))
   
-	text_chunks = generate_chunks_from_dataset(subset, chunk_size=512)
+  text_chunks = generate_chunks_from_dataset(subset, chunk_size=512)
```

Comment:
Nice catch on the spacing here. Think there's a similar spacing isue for the `dataset=load_from_disk` spacing issue too